### ISSUE TITLE
Settings Dialog not following selected application theme.

### ISSUE THREAD

**Issue by DomeAlessi** [snoopwpf/snoopwpf#456]
Hello!
After changing a color scheme from the Theme dropdown and opening the Snoop Settings Dialog from the main widget UI button the overall application style is not applied to the Settings Dialog.

The last saved them will be applied to the Settings dialog at startup.

Would it be possible to make the Settings Dialog follow the current application theme?

**Comment by batzen** [comment]
It would be possible but would require either a filesystem watcher or inter process communication.

Wouldn't it be better if you could choose the theme from the Snoop main app instead?
If that's the case that would be ideal for the planned re-design of the main App.

**Comment by DomeAlessi** [comment]
Yeah it would be perfect and it feels like it would make more sense.
I tried to take a look at the code and in effect it would take a some refactoring, I tried to move the SettingsView into Snoop.Core with the other view but i have different instances of Snoop.Core.Settings() created (although they should be unique) but if you plan to refactor the main view I will abandon this lead and gladly take your proposal to move them in the main App.
